Source/JavaScriptCore/yarr/YarrParser.h

```diff
@@ -47,13 +48,75 @@ enum class CharacterClassSetOp : uint8_t {
     Subtraction
 };
 
+template <class T> concept YarrSyntaxCheckable = requires (T& checker, Vector<Vector<char32_t>>& disjunctionStrings, const String& subpatternName) {
```

Comment:
https://webkit.org/code-style-guidelines/ still does not have a mention about to writing a C++ concept.

Instead, I referred some usages of C++ concepts in our code and make here similar to it, but I seem style-checker would say some errors if I follow it.
https://github.com/weinig/WebKit/blob/0cf319253661cb3069aaf52386cb55389eb1fcb6/Source/WTF/wtf/text/StringConcatenate.h#L556-L558

Should I add this file to the ignore list of style cheker? (I think it's not good choice because this file is too large to ignore style checking).